src/main/java/lotto/domain/LottoTicketCollection.java

```diff
@@ -0,0 +1,26 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class LottoTicketCollection {
+
+    private final List<LottoTicket> autoTickets;
+    private final List<LottoTicket> manualTickets;
+
+    public LottoTicketCollection(List<LottoTicket> autoTickets, List<LottoTicket> manualTickets) {
+        this.autoTickets = autoTickets;
+        this.manualTickets = manualTickets;
+    }
+
+    public List<LottoTicket> getAutoTickets() {
+        return autoTickets;
+    }
+
+    public List<LottoTicket> getManualTickets() {
+        return manualTickets;
+    }
+
+    public List<List<LottoTicket>> getAllTicketCollection() {
+        return List.of(manualTickets, autoTickets);
+    }
```

Comment:
`List<List<LottoTicket>>` 리스트안에 리스트가 있기 때문에 복잡도가 높아지게됩니다.

수동/자동 상관없이 결국 같은 로또이기 때문에 `getAllTicket` 이면 합쳐서 내보내는 게 좋을 것 같아요

```suggestion
     public List<LottoTicket> getAllTicketCollection() {
        return Stream.concat(
                autoTickets.stream(),
                manualTickets.stream()
        ).collect(Collectors.toList());
    }
```